packages/angular/test/base/src/app/test/test.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-test',
+  templateUrl: './test.component.html',
+  standalone: true
+})
+export class TestComponent {
+  constructor() {
+    console.log('This should be undefined:', customElements.get('ion-app'));
```

Comment:
This is a placeholder for now to verify that the IonicModule is not being defined